documentation/service-rules/ecf1/participants.md

```diff
@@ -0,0 +1,567 @@
+
+## Contents
+
+[Participant Types](#participant-types)
+
+[Participant Interactions](#participant-interactions)
+
+[API Access Conditions](#api-access-conditions)
+
+[Participant Attributes](#participant-attributes)
+
+[API Endpoints](#api-endpoints)
+
+[User Access](#user-access)
+
+[Error Handling](#error-handling)
+
+[Participant Journeys](#participant-journeys)
+
+[Technical Reference](#technical-reference)
+
+[Glossary](#glossary)
+
+## Participant Types
+
+The ECF service supports two types of participants:
+
+### Early Career Teachers (ECTs)
+
+ECTs are teachers in the first two years of their career who receive structured mentoring and support through the ECF programme. They are the primary beneficiaries of the programme and have specific attributes and statuses that track their progress.
+
+### Mentors
+
+Mentors are experienced teachers who provide guidance and support to ECTs. They are also registered in the system and have their own set of attributes and statuses.
+
+Both participant types follow similar registration processes but may have different attributes and relationships within the system.
+
+## Participant Interactions
+
+### School Induction Tutor Registration
+
+The participant journey begins when a school induction tutor registers an ECT or Mentor in the system. The registration process involves:
+
+1. The induction tutor enters the participant's details, including:
+   - Full name
+   - Email address
+   - Teacher Reference Number (TRN)
+   - School details
+
+2. The system validates the information, particularly the TRN, to ensure it corresponds to a registered teacher.
+
+3. Upon successful validation, the participant is created in the system with an initial status of "active".
+
+### Partnership Requirements
+
+For participants to be visible via the API, a partnership must exist between:
+
+- The school where the participant is registered
+- A lead provider who delivers the training
+
+This partnership establishes the relationship necessary for data sharing and training delivery.
+
+### Lead Provider Relationship
+
+The lead provider must have one of the following relationships with the school:
+
+- Be the default partnership for the school
+- Have an established relationship with that school
+
+Without this relationship, participants will not be visible through the API.
+
+## API Access Conditions
+
+To successfully retrieve participants via the API, all three of the following conditions must be met:
+
+1. **Partnership Formation**: A partnership must be formed between the lead provider and the school.
+2. **Participant Registration**: The participant must be registered in the system by a school induction tutor.
+3. **Lead Provider Relationship**: The lead provider must be either:
```

Comment:
can we make sure we say that challenged partnerships don't show participants, or they disappear if a partnership is challenged?